### PR TITLE
fix-945-measureBoundingBox

### PR DESCRIPTION
**Pull request by hrgdavor**
fix for #945 checks transformed bounding box and swaps values to make sure  max values are in second vec3

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

### CODE REVIEW

File: packages/modeling/src/measurements/measureBoundingBox.js [hunk 1]
```diff
@@ -175,9 +175,21 @@ const measureBoundingBoxOfGeom3 = (geometry) => {
   return boundingBox
 }
 
+const fixBound = (i, v1,v2)=>{
```
Comment (z3dev) [inline packages/modeling/src/measurements/measureBoundingBox.js:178]: I don’t think that anyone will understand this function in one or two years. Please add a comment.

Comment (hrgdavor) [reply packages/modeling/src/measurements/measureBoundingBox.js:179]: :D np, let me know if the new comments are sufficient

**Review by z3dev — DISMISSED** [review]
@hrgdavor these changes look fine. thanks.

any chance to get a test case?

**Comment by hrgdavor** [comment]
> any chance to get a test case?

ofc, thnx for the reminder, I keep forgetting to add tests ... just not used to doing it 😄

**Review by z3dev — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**